docs/guide/installation.rst

```diff
@@ -0,0 +1,100 @@
+************
+Installation
+************
+
+Requirements
+============
+
+These are the minimum versions of packages needed to install sunpy:
+
+- python 3.7
+- astropy 4.0.3
+- scipy 1.2
+- parfive 1.1
+
+Installing Scientific Python and sunpy
+======================================
+
+sunpy is part of the wider ecosystem of scientific Python packages for solar physics.
+Therefore a working sunpy installation is more about installing the scientific Python ecosystem than sunpy itself.
+
+If you do not currently have a working scientific Python distribution this guide will set you up with the Miniconda, which makes it easy to install and manage your scientific Python packages.
+
+To install the Miniconda Python distribution follow the instructions at
+`here <https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html>`__.
+Although Miniconda makes it simple to switch between Python versions, we recommend that new users install
+the latest Python 3.x version of Miniconda as SunPy only supports Python 3.7+.
+
+The reason we choose Miniconda over Anaconda, is mainly due to the size as Anaconda comes with a full install of packages you probably do not need and this way you have more direct control over what has been installed into your Python virtual environment.
+Furthermore, you bypass the need for the conda resolver to sort out your root environment which should make "conda" faster to use.
+
+Installing sunpy using Miniconda
+--------------------------------
+
+To install SunPy launch a system command prompt or the 'Anaconda Prompt' (under Windows).
+First configure conda for to add the `conda-forge channel <https://conda-forge.org/>__`::
+
+    conda config --add channels conda-forge
+    conda config --set channel_priority strict
+
+and now to install sunpy::
+
+    conda install sunpy
```

Comment:
Should we be recommending people start a fresh env for sunpy? Is that too far for the basic install docs?

Reply:
I thought this was aimed at people without a previous sunpy install. Who are installing it for the first time.

Reply:
Sure, I just mean changing this to `conda create -n sunpy python sunpy; conda activate sunpy`

Reply:
(I am not sure this is a good idea, but in general using a venv is always good right?)

Reply:
I will redraft this section further then.

Reply:
Fixed.

Reply:
I would be interested to know other peoples opinions on this, and we might need to make it clearer that you have to activate the env each time. Then does our install docs just become a conda guide :exploding_head: 

Maybe we should actually just give people the install into the root env command and link them to a environments guide?

Reply:
I definitely think its good to do a 'best practice' but for a lot of users, however we should probably be aware that many scientific users, especially those not fully accustomed to environments etc, and basically the majority of the coding will use sunpy etc and just want all the required packages in their base env we should probably have a simple method for doing this.